src/ledger/rocksdb.zig

```diff
@@ -70,14 +70,9 @@ pub fn RocksDB(comptime column_families: []const ColumnFamily) type {
             };
         }
 
-        pub fn deinit(self: *Self, delete_disk: bool) void {
+        pub fn deinit(self: *Self) void {
```

Comment:
any reason you removed `delete_disk` ?

Reply:
Whenever I see a bool as a function parameter, I ask myself if it would be better to actually have two separate functions instead of one function with a bool. In this case, I think it would be cleaner to have two separate functions.

There is no need currently to delete the data from the disk so I didn't add the other function.

Also the recommended way to delete rocksdb from the disk is by using the `DestroyDB` function. Today I added this function to rocksdb-zig and created a new branch in sig that upgrades to use this version of the library.